src/site/content/en/lighthouse-performance/unused-javascript/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Remove unused JavaScript
+description: |
+  Learn how to pass Lighthouse's "Remove unused JavaScript" audit.
+web_lighthouse:
+  - unused-javascript
+date: 2020-06-17
+---
+
+Unused JavaScript can slow down your page speed if the JavaScript is
+[render-blocking][crp]. When the browser encounters a render-blocking script
+it has to download, parse, compile, and evaluate the script before it
+can proceed with all of the other work that's needed for rendering the page.
+
+{% Aside %}
+  Since the JavaScript code that Lighthouse flags in this audit is unused,
+  there is no evaluation cost. However, there are download, parse, and compilation
+  costs.
+{% endAside %}
+
+Even if the unused JavaScript is not render-blocking, sending unused code over
+the network is always a waste of bandwidth, which is especially important
+for mobile users that don't have unlimited data plans.
```

Comment:
```suggestion
Unused JavaScript has network and runtime costs:

* TODO

can slow down your page speed if the JavaScript is
[render-blocking][crp]. When the browser encounters a render-blocking script
it has to download, parse, compile, and evaluate the script before it
can proceed with all of the other work that's needed for rendering the page.

{% Aside %}
  Since the JavaScript code that Lighthouse flags in this audit is unused,
  there is no evaluation cost. However, there are download, parse, and compilation
  costs.
{% endAside %}

Even if the unused JavaScript is not render-blocking, sending unused code over
the network is always a waste of bandwidth, which is especially important
for mobile users that don't have unlimited data plans.
```